src/it/scala/net/snowflake/spark/snowflake/TruncateTableSuite.scala

```diff
@@ -384,7 +384,10 @@ class TruncateTableSuite extends IntegrationSuiteBase {
   // 2. Configure truncate_table = on and usestagingtable=off can workaround this issue.
   test("write table with different schema") {
     val accountName = System.getenv(SNOWFLAKE_TEST_ACCOUNT)
```

Comment:
To reviewers:
This fix only skips to run the test case for released SC back-compatibility regress test.
The test is still included in IT test in Github Action and Travis.